.github/workflows/test.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           - openssl
   hw:
     name: hw ${{ matrix.toolchain }} ${{ matrix.profile.name }} ${{ matrix.features }}
-    runs-on: [self-hosted, linux, sev]
+    runs-on: [self-hosted, linux, snp]
```

Comment:
I think this should be merged into the first commit that started testing the SNP ioctls.